polaris/tasks/ocean/external_gravity_wave/forward.py

```diff
@@ -0,0 +1,302 @@
+import time as time
+
+from polaris.ocean.convergence.spherical import SphericalConvergenceForward
+from polaris.ocean.model import OceanModelStep, get_time_interval_string
+
+
+class Forward(SphericalConvergenceForward):
+    """
+    A step for performing forward ocean component runs as part of the external
+    gravity wave test case
+
+    Attributes
+    ----------
+    do_restart : bool
+        Whether this is a restart run
+    """
+
+    def __init__(
+        self,
+        component,
+        name,
+        subdir,
+        mesh,
+        init,
+        refinement_factor,
+        refinement,
+        dt_type,
+        do_restart=False,
```

Comment:
I don't think it makes sense to have `do_restart` be defined as an attribute in the constructor because it would be difficult for users to change. Correct me if I'm missing something about how this works.